### ISSUE TITLE
[bitnami/kafka] don't start with zookeeper mode, it try to initialize kraft

### ISSUE THREAD

**Issue by cloud-66** [bitnami/charts#16275]
### Name and Version

bitnami/kafka 22.0.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. install kafka with zookeeper

2. After install 
```console
$ k logs -f pod/kafka-2
Defaulted container "kafka" out of: kafka, auto-discovery (init)
kafka 11:29:09.53
kafka 11:29:09.54 Welcome to the Bitnami kafka container
kafka 11:29:09.54 Subscribe to project updates by watching https://github.com/bitnami/containers
kafka 11:29:09.54 Submit issues and feature requests at https://github.com/bitnami/containers/issues
kafka 11:29:09.54
kafka 11:29:09.54 INFO  ==> ** Starting Kafka setup **
kafka 11:29:09.63 DEBUG ==> Validating settings in KAFKA_* env vars...
kafka 11:29:09.63 WARN  ==> KAFKA_CFG_LISTENERS must include a listener for CONTROLLER
kafka 11:29:09.64 WARN  ==> You set the environment variable ALLOW_PLAINTEXT_LISTENER=yes. For safety reasons, do not use this flag in a production environment.
kafka 11:29:09.66 INFO  ==> Initializing Kafka...
kafka 11:29:09.67 INFO  ==> No injected configuration files found, creating default config files
kafka 11:29:10.35 INFO  ==> Configuring Kafka for inter-broker communications with PLAINTEXT authentication.
kafka 11:29:10.35 WARN  ==> Inter-broker communications are configured as PLAINTEXT. This is not safe for production environments.
kafka 11:29:10.36 INFO  ==> Configuring Kafka for client communications with PLAINTEXT authentication.
kafka 11:29:10.37 WARN  ==> Client communications are configured using PLAINTEXT listeners. For safety reasons, do not use this in a production environment.
kafka 11:29:10.37 INFO  ==> Configuring Kafka for external client communications with PLAINTEXT authentication.
kafka 11:29:10.37 WARN  ==> External client communications are configured using PLAINTEXT listeners. For safety reasons, do not use this in a production environment.
kafka 11:29:10.38 INFO  ==> Initializing KRaft...
kafka 11:29:10.38 WARN  ==> KAFKA_KRAFT_CLUSTER_ID not set - If using multiple nodes then you must use the same Cluster ID for each one
kafka 11:29:12.14 INFO  ==> Generated Kafka cluster ID 'FlAqud5dRwqT--FpikNdug'
kafka 11:29:12.14 INFO  ==> Formatting storage directories to add metadata...
Exception in thread "main" org.apache.kafka.common.config.ConfigException: You must set `node.id` to the same value as `broker.id`.
        at kafka.server.KafkaConfig.validateValues(KafkaConfig.scala:2112)
        at kafka.server.KafkaConfig.<init>(KafkaConfig.scala:2107)
        at kafka.server.KafkaConfig.<init>(KafkaConfig.scala:1533)
        at kafka.tools.StorageTool$.$anonfun$main$1(StorageTool.scala:41)
        at scala.Option.flatMap(Option.scala:271)
        at kafka.tools.StorageTool$.main(StorageTool.scala:41)
        at kafka.tools.StorageTool.main(StorageTool.scala)
```

3. it looks like it is using KRAFT even i have disabled it
How to install kafka with zookeeper ?

4 screen

![image](https://user-images.githubusercontent.com/41143306/235300503-4967335c-71ba-428b-b9f2-769bcfb04595.png)

### Are you using any custom parameters or values?

kafka-values.yml
```yaml
image:
  debug: true
replicaCount: 3
tolerations: 
   - key: "node-role.kubernetes.io/control-plane"
     operator: "Equal"
     value: ""
     effect: "NoSchedule"
kraft:
  enabled: false
externalAccess:
  enabled: true
  autoDiscovery:
    enabled: true
  service:
    type: NodePort
    nodePorts:
    - 30001
    - 30002
    - 30003
zookeeper:
  enabled: true
  replicaCount: 3
```

### What is the expected behavior?

run kafka with zookeeper

### What do you see instead?

kafka try to use kraft and get exception, kafka don't start

**Comment by cloud-66** [comment]
Needs to add but from my point of view it's better to add this to template
```
extraEnvVars:
  - name: KAFKA_ENABLE_KRAFT
    value: "false"
```

**Comment by carrodher** [comment]
Thanks for creating this issue and the associated PR. The team will review it and provide feedback. Once merged the PR, this issue will be automatically closed.

**Comment by cloud-66** [comment]
merged #16276
